frontend/src/components/AppButton.vue

```diff
@@ -1,7 +1,16 @@
 <template>
-    <button type='button' :class="alt ? 'standard-button' : 'alt-button'">
-        <slot></slot>
-    </button>
+    <div>
```

Comment:
This seems like two components.  Else, one component but oddly shaped.

Reply:
It's one and oddly shaped, just wanted something that worked. Not sure about the proper pattern here.